docs/pages/versions/unversioned/sdk/updates.md

```diff
@@ -3,89 +3,127 @@ title: Updates
+
+<TableOfContentSection title='Related Types' contents={['EventSubscription', 'UpdateEvent', 'UpdateEventType']} />
+
+<TableOfContentSection title='Error Codes' contents={[]} />
+
+## Constants
+
+### `Updates.isEmergencyLaunch`
+
+(_boolean_) `expo-updates` does its very best to always launch monotonically newer versions of your app so you don't need to worry about backwards compatibility when you put out an update. In very rare cases, it's possible that `expo-updates` may need to fall back to the update that's embedded in the app binary, even after newer updates have been downloaded and run (an "emergency launch"). This boolean will be `true` if the app is launching under this fallback mechanism and `false` otherwise. If you are concerned about backwards compatibility of future updates to your app, you can use this constant to provide special behavior for this rare case.
+
+### `Updates.manifest`
+
+(_object_) The [manifest](../../workflow/how-expo-works/#expo-development-server) object for the update that's currently running.
+
+## Methods
 
-Immediately reloads the current experience. This will use your app.json `updates` configuration to fetch and load the newest available JS supported by the device's Expo environment. This is useful for triggering an update of your experience if you have published a new version.
+### `Updates.reloadAsync()`
 
-### `Updates.reloadFromCache()`
+Instructs the app to reload using the most recently downloaded version. This is useful for triggering a newly downloaded update to launch without the user needing to manually restart the app.
 
-Immediately reloads the current experience using the most recent cached version. This is useful for triggering an update of your experience if you have published and already downloaded a new version.
+This method cannot be used in development mode, and the returned `Promise` will be rejected if you try to do so.
+
+#### Returns
+
+A `Promise` that resolves right before the reload instruction is sent to the JS runtime, or rejects if it cannot find a reference to the JS runtime.
+
+If the `Promise` is rejected in production mode, it most likely means you have installed the module incorrectly. Double check you've followed the instructions above. In particular, on iOS ensure that you set the `bridge` property on `EXUpdatesAppController` with a pointer to the `RCTBridge` you want to reload, and on Android ensure you either call `UpdatesController.initialize` with the instance of `ReactApplication` you want to reload, or call `UpdatesController.setReactNativeHost` with the proper instance of `ReactNativeHost`.
 
 ### `Updates.checkForUpdateAsync()`
 
-Check if a new published version of your project is available. Does not actually download the update. Rejects if `updates.enabled` is `false` in app.json.
+Checks the server at the provided remote URL to see if a newly deployed version of your project is available. Does not actually download the update.
```

Comment:
"provided remote URL" makes it sound like you can specify the URL. Reword so that it's clearer what URL this is referring to.